File: Python/baekjoon/단계별로 풀어보기/문자열/2941_크로아티아 알파벳.py
# https://www.acmicpc.net/problem/2941

p = ['c=', 'c-', 'dz=', 'd-', 'lj', 'nj', 's=', 'z=']
s = input()
for x in p:
    s = s.replace(x, 'a')
print(len(s))

File: C#/Baekjoon/심화1/2941_크로아티아 알파벳.cs
//https://www.acmicpc.net/problem/2941

string s = Console.ReadLine();
string[] a = new string[8] { "c=", "c-", "dz=", "d-", "lj", "nj", "s=", "z=" };
foreach (string c in a)
{
    s = s.Replace(c, "1");
}
Console.WriteLine(s.Length);